dm/dm/config/task_converters.go

```diff
@@ -166,10 +166,7 @@ func OpenAPITaskToSubTaskConfigs(task *openapi.Task, toDBCfg *DBConfig, sourceCf
 			}
 			subTaskCfg.Meta = meta
 		}
-		// check must set meta when mode is ModeIncrement
-		if subTaskCfg.Meta == nil && subTaskCfg.Mode == ModeIncrement {
-			return nil, terror.ErrConfigMetadataNotSet.Generate(i, ModeIncrement)
```

Comment:
why delete this?

Reply:
this is to avoid error when there is no binlog postions in incremental mode, so user can start task by conditions such as  `start_time`. this is a requirement by @sunzhaoyang .